src/main/java/gregtech/api/gui/resources/TextTexture.java

```diff
@@ -42,11 +52,18 @@ public TextTexture setDropShadow(boolean dropShadow) {
 
     public TextTexture setWidth(int width) {
         this.width = width;
-        if (FMLCommonHandler.instance().getSide().isClient()) {
-            if (this.width > 0) {
-                texts = Minecraft.getMinecraft().fontRenderer.listFormattedStringToWidth(text, width);
-            } else {
-                texts = Collections.singletonList(text);
+        return this;
+    }
+
+    public TextTexture setText(String text) {
+        if (this.text.isEmpty() || !this.text.equals(text)) {
```

Comment:
`this.text.isEmpty()` skips the case where the incoming `text` parameter is also empty, in which case nothing needs to be set.

Reply:
tbh the `isEmpty()` can be removed. the not equals check should be enough